packages/common/pipes/parse-date.pipe.ts

```
@@ -0,0 +1,74 @@
+      return this.options.default
+        ? this.options.default()
+        : (value as undefined | null);
+    }
+
+    if (!value) {
+      throw this.exceptionFactory('Validation failed (no Date provided)');
+    }
+
+    const transformedValue = new Date(value);
+
+    if (isNaN(transformedValue.getTime())) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)